setup.cfg

```diff
@@ -10,7 +10,7 @@ skip_covered = True
 max-line-length = 90
 max-complexity = 10
 exclude = *migrations*
-ignore = D100,D104,D203,D204
+ignore = D100,D104,D203,D204,C814,C815,C816
```

Comment:
This ignores warnings for:

- Python 2, which we do not support.
- Python 3.5+, and Python 3.6+. When we drop versions before these, we can tighten up the slack.